Terraform_modules/rds_serverless_cluster/variables.tf

```diff
@@ -50,6 +50,19 @@ variable "database_serverlessv2_scaling_max_capacity" {
   description = "This sets the maximum scaling capacity of the severless database in Aurora capacity units (ACU)."
 }
 
+
+variable "backup_retention_period" {
+  type        = number
+  default     = 1
```

Comment:
There is a limit on the values allowed - there should be some validation added... https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/db_instance#backup_retention_period